### PR TITLE
Fixes: Arabic frameit imagemagick issue #7522 and similar

### PR DESCRIPTION
**Pull request by iMokhles**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

##### Describe your changes in detail

add rechaper.rb from [ShadyAbuKalam/Ruby-Arabic-Reshaper](https://github.com/ShadyAbuKalam/Ruby-Arabic-Reshaper)
and added it to frameit to reshape arabic text 

##### Why is this change required? What problem does it solve?
solves wrong arabic texts encoding from imagemagick 

##### If it fixes an open issue, please link to the issue here.
[issue #7522](https://github.com/fastlane/fastlane/issues/7522)

##### Please describe in detail how you tested your changes
used MindNode frameit example from [MindNode/screenshots](https://github.com/fastlane/examples/tree/master/MindNode/screenshots)

### CODE REVIEW

File: sigh/lib/assets/resign.sh [hunk 1]
```diff
@@ -536,7 +536,7 @@ function resign {
         log "Updating the bundle version (CFBundleVersion) from '$CURRENT_VALUE' to '$BUNDLE_VERSION'"
         PlistBuddy -c "Set :CFBundleVersion $BUNDLE_VERSION" "$APP_PATH/Info.plist"
     fi
-
+
```
Comment (hjanuschka) [inline sigh/lib/assets/resign.sh]: that happend by accident?

**Review by hjanuschka — CHANGES_REQUESTED** [review]
i really would love to see #7522  fixed.
however i am unsure about the introduction to a new dep.

also there are a couple of rubocop issues,  that would need to get addressed.

@KrauseFx, @mfurtak  hows your view on the introduction of a new dependencie?

File: sigh/lib/assets/resign.sh (hunk 1 above)
Comment (iMokhles) [inline sigh/lib/assets/resign.sh]: i had modified version to sign dylibs in main bundle pull request #6364 so i set it to the original version

**Comment by hjanuschka** [comment]
@iMokhles you see any chance to find a solution without the new gem dependencies? (as most of the fastlane users are not in the need of arabic letters, i guess) and bringing in new deps. is always crucial.

**Comment by KrauseFx** [comment]
Agreed with @hjanuschka, we can't introduce a new dependency for this unfortunately

**Comment by mfurtak** [comment]
I'm going to close this for now, but we can always reopen it if we can make more forward progress :+1:

**Pull request closed without merging**